packages/core/src/lib/toast/toast-banner.svelte

```diff
@@ -0,0 +1,119 @@
+<svelte:options immutable />
+
+<script lang="ts">
+import cx from 'classnames';
+import { createEventDispatcher } from 'svelte';
+import { Button, Icon, type IconName } from '$lib';
+import { ToastVariant, type ToastVariantType } from './variants';
+
+/** The type of toast banner to be displayed to the user */
+export let variant: ToastVariantType;
+
+/** Whether there is a close icon displayed */
+export let closeable: boolean | undefined = false;
+
+/** Whether there is an action button displayed */
+export let action = '';
```

Comment:
Avoid using empty strings as falsey values; use `undefined` to declare that something might not be specified. It's a lot easier to statically analyze